Navigation Bar Changes - Dropdown Menu

Louie112 opened the pull request:
Consolidated the page navigation options into a dropdown menu.

Louie112 committed:
dropdown menu

oompas reviewed (changes requested):
- Put the home button separate from the drop down
- Put some sort of divider like a | type of line before the user's name with the respective drop down
- For a learner, the nav bar should look like: "Home | Profile Name"
- For an admin, the nav bar should look like: "Home, Admin Tools | Profile Name" (comma just there for organizing my thoughts, not for actual design choice)

Essentially, this makes it more intuitive that the profile name drop down allows the user to view their own profile and log out of the platform. After you make these changes, the PR should be good to merge after another review!

Louie112 committed:
fixes

changed dropdown title to name of user.
added a divider between dropdown and other options

Louie112 committed:
Update page.tsx

Louie112 committed:
Update page.tsx

Louie112 committed:
Merge branch 'staging' into dropdown_menu

GeeCracker reviewed (changes requested):
see discord

oompas commented:
Summary of tonight's team discussion on the issue:
Since the platform is desktop only and we weren't able to implement the optional requirement for user profile pictures, it's most likely best to scrap these changes since there will only be 4 options in the navigation bar for an administrator and 3 options for a learner.
